src/dpq2/conv/to_d_types.d

```diff
@@ -47,7 +47,7 @@ private alias ET = ConvExceptionType;
 
 /// Returns cell value as native string based type from text or binary formatted field
 T as(T)(in Value v) pure @trusted
-if(is(T : string))
+if(is(T : string) || is(T == Nullable!string))
```

Comment:
`is(T == Nullable!string)` isn't needed here as T:string already catches that

Reply:
Reverted